app/views/welcome/_map.html.slim

```diff
@@ -42,35 +41,30 @@ css:
 
 javascript:
     // initialize the map
-    var map = L.map('map', { zoomControl: #{defined?(zoomControl) ? zoomControl : true } }).setView([52.511946, 13.406166], 10);
-    var imgNotAvailable = 'https://www.postfreeadvertising.com/wp-content/uploads/2018/08/no-image-available-grid-263x172.png'
-    var toBeUsedTile = 'https://api.mapbox.com/styles/v1/mapbox/light-v10/tiles/{z}/{x}/{y}?access_token={accessToken}'
-    var stadiamaps = 'https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}.png'
-    var osmMap = 'https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png'
-    var cartoDB = 'https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}{r}.png'
+    var map = L.map('map', { zoomControl: #{defined?(zoomControl) ? zoomControl : true } }).setView([52.511946, 13.406166], 12);
+    var myStyle = 'https://api.mapbox.com/styles/v1/myusernameisthis/ck8k8w25g02d01jmnnbehc46d/tiles/512/{z}/{x}/{y}?access_token={accessToken}'
```

Comment:
"myusernameisthis" --> is this the actual username?

Reply:
haha, yes, my username on mapbox :D

Reply:
ok for now, but we should keep this in a seperate file. rails has a module for this called "credentials"